packages/dropdowns/src/Dropdown/Dropdown.tsx

```diff
@@ -212,22 +212,22 @@ const Dropdown: React.FunctionComponent<IDropdownProps> = props => {
         }}
         {...downshiftProps}
       >
-        {downshift => (
-          <DropdownContext.Provider
-            value={{
-              itemIndexRef,
-              previousItemRef,
-              previousIndexRef,
-              nextItemsHashRef,
-              popperReferenceElementRef,
-              selectedItems,
-              downshift: transformDownshift(downshift),
-              containsMultiselectRef
-            }}
-          >
-            {children}
-          </DropdownContext.Provider>
-        )}
+        {downshift => {
```

Comment:
I've moved the context `value`'s to not be inline in an attempt to reduce unnecessary renders/positions.